### ISSUE TITLE
error accessing

### ISSUE THREAD

**Issue by bonj1987** [Cacti/plugin_reportit#102]
i just installed this on 1.2.23. followed the directions.. but i get this when trying to go to mgmt\reports
Sorry, we could not process your last action.
Error:	401 Permission Denied
Reason:	You are not permitted to access this section of Cacti. If you feel that this is an error. Please contact your Cacti Administrator.

the instructions say to go to utilities\user mgmt and adjust your realm.. i think that should read configuration\users
that said.. im logged in as an admin and have access to everything in cacti, but keep getting that error.

**Comment by bonj1987** [comment]
can anyone take a look at this for me? let me know if i need to do anything to fix this?

**Comment by TheWitness** [comment]
Show us the user "Permissions Tab".

**Comment by bonj1987** [comment]
![USERPERMS](https://user-images.githubusercontent.com/115038949/220735985-9d0e0ab8-ec19-4785-9e73-4bf3d23c2d05.JPG)
Here you go. user perms for the admin account, which I am using.

**Comment by lutfisan** [comment]
![image](https://user-images.githubusercontent.com/29010489/220874673-e08f9244-b18f-43b4-be3f-ef2069736c63.png)
identical mistake occurs here.

**Comment by ck-australia** [comment]
Did you ever get this sorted Im having the same issue

**Comment by bonj1987** [comment]
nope

**Comment by TheWitness** [comment]
@browniebraun is the author.  He needs to chime in.

**Comment by TheWitness** [comment]
Does your plugin_realms table look similar to this?

```sql
MariaDB [cacti]> select * from plugin_realms where plugin='reportit';
+-----+----------+--------------------------------------------+----------------------+
| id  | plugin   | file                                       | display              |
+-----+----------+--------------------------------------------+----------------------+
| 160 | reportit | view.php,charts.php                        | View Reports         |
| 161 | reportit | reports.php,rrdlist.php,items.php,run.php  | Create Reports       |
| 162 | reportit | templates.php,measurands.php,variables.php | Administrate Reports |
+-----+----------+--------------------------------------------+----------------------+
```

**Comment by ck-australia** [comment]
No I have nothing related to ReportIT in that table at all.

**Comment by TheWitness** [comment]
That means that the plugin did not install properly.  What version of the plugin are you using?

**Comment by TheWitness** [comment]
Show up on the plugin page like this for me.

![image](https://user-images.githubusercontent.com/1439914/223305040-f77911cb-4a8d-48d2-a120-6eae7f0dcf99.png)

**Comment by ck-australia** [comment]
Plugin version is 1.1.3

**Comment by TheWitness** [comment]
Upload a screen shot.  Then, if it's a fresh install, uninstall and re-install it.  Look to the Cacti log immediately and look for errors.

**Comment by ck-australia** [comment]
![image](https://user-images.githubusercontent.com/77236712/223308514-00cdc0f6-d9a1-451c-9b73-a2189cef977e.png)
Will remove and reinstall it and post any errors

**Comment by ck-australia** [comment]
Removed and reinstalled it with no errors at all. 
![image](https://user-images.githubusercontent.com/77236712/223308795-a5d73efa-22d9-4d14-aa64-a84ca6c75306.png)

**Comment by bonj1987** [comment]
i ran that same query.. came up with nothing.. i did a select from the plugin_realms table.. doesnt even show reportit
![plugins](https://user-images.githubusercontent.com/115038949/223433108-9877a88e-d2cd-4bd8-9ad4-86e46d0985ce.JPG)

**Comment by TheWitness** [comment]
I'll see what I can do tonight.  This issue does not seem something that @browniebraun needs to intervene on.  It appears to be plugin install/uninstall related.  Been trying to take some time away from Cacti to recharge the batteries.

**Comment by bonj1987** [comment]
any update by chance?

**Comment by TheWitness** [comment]
I'm looking into it tonight.  I know what happened, just need to fine out where and why now.

**Comment by TheWitness** [comment]
Try re-installing again.  It should be working now.

**Comment by ck-australia** [comment]
Confirmed all good mate cheers

**Comment by TheWitness** [comment]
Screen shots?

**Comment by bonj1987** [comment]
confirmed i can get into it. now i have to figure out how to create the reports. the documention isnt very clear.. for example, if i want to create a report showing disk space on a group of servers, i added the graphs to a report, but when i run it i get errors that the RRD files dont exist. i can throw up a screen shot if you like.

**Comment by syrus1006** [comment]
Hello Everyone! I have a very similar issue to the one above. I have 1.2.25 Cacti version installed, and Reportit 1.1.2. 
When I click on "Report" under the "Templates" menu, I get the "401 permission denied" error.
Also in the logs, I get several "PLUGIN WARNING: Plugin 'reportit' is attempting to call 'api_plugin_register_realm' improperly in function 'reportit_config_arrays' messages.

I tried to reinstall the plugin, but the issue is still there. Also, I can see that others have Reportit version 1.1.3., but I could not find this version anywhere.

**Comment by xmacan** [comment]
It was fixed in 1.1.3, closing
